pkg/api/api.go

```diff
@@ -251,6 +251,9 @@ func (hs *HTTPServer) registerRoutes() {
 		apiRoute.Get("/plugins", Wrap(hs.GetPluginList))
 		apiRoute.Get("/plugins/:pluginId/settings", Wrap(GetPluginSettingByID))
 		apiRoute.Get("/plugins/:pluginId/markdown/:name", Wrap(GetPluginMarkdown))
+		apiRoute.Get("/plugins/:pluginId/health", Wrap(hs.CheckHealth))
```

Comment:
Does it make sense to add an api endpoint now or later?
```
apiRoute.Get("/plugins/:pluginId/api", Wrap(hs.GetResourceApi))
```
It will always return a JSON object.

It does not yet need to be in the SDK, but not sure if easier to add now or later